src/pages/Details.js

```diff
@@ -2,83 +2,82 @@ import React from 'react';
 import '../styling/Details.css'
 import Source from '../components/Source'
 import { dateToString } from '../util';
-import Carousel from 'react-bootstrap/Carousel' 
+import Carousel from 'react-bootstrap/Carousel'
 import logo from '../images/cwb-logo-reverse-w-tagline 1.png';
 import axios from 'axios';
 import { withRouter } from 'react-router-dom';
-
-
+ 
+ 
 class Details extends React.Component {
-   componentDidMount() {
-    const { match: { params } } = this.props;
-  
+  componentDidMount() {
+   const { match: { params } } = this.props;
     axios.get(`/details/${params.userId}`)
-      .then(({ data: user }) => {
-        console.log('user', user);
-  
+     .then(({ data: user }) => {
```

Comment:
for my understanding: what does this do? is it just logging in users

Reply:
I think this allows for the url to be unique, looking at the whole code blurb

`componentDidMount() {
   const { match: { params } } = this.props;
    axios.get(`/details/${params.userId}`)
     .then(({ data: user }) => {
        this.setState({ user });
 
     });
 }`